import/popups/WebrtcPermissionDialog.qml

```diff
@@ -0,0 +1,174 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC.
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+UserPromptDialog {
+    id: dialog
+
+    property alias devices: webrtc.devices
+    property alias origin: webrtc.origin
+    property alias rememberValue: webrtc.rememberValue
+    property alias choices: webrtc.choices
+
+    property alias acceptText: webrtc.acceptText
+    property alias cancelText: webrtc.cancelText
```

Comment:
Are the `acceptText` and `cancelText` aliases needed, or can they be dropped? I didn't see them being used.

Reply:
I wasn't sure, I copied the same from LocationDialog.qml. It seems they are not needed at all.